sample/sample.md

```diff
@@ -0,0 +1,175 @@
+Neomerx JSON API sample application (basic usage)
```

Comment:
- Sample output is lengthy
- User can view it with `php sample.php`
- Better to leave short description otherwise visitors might be confused